origin-linking/src/logic/hot.js

```diff
@@ -13,7 +18,23 @@ class Hot {
     origin.contractService.transactionSigner = this.account.signTransaction
   }
 
+  isAllowedCall(cmd, params){
```

Comment:
Can you add a comment explaining the minFee logic ? Not knowing much about this it is not clear why this is needed...